sdk/rust/oak/src/io/mod.rs

```diff
@@ -49,6 +49,20 @@ pub fn node_create<T: Encodable + Decodable>(
     Ok(sender)
 }
 
+/// Creates a node and corresponding inbound channel of the same type, for nodes that are
+/// instantiated via the [`crate::entrypoint_command_handler`] macro.
```

Comment:
Could there be a situation where `impl ::oak::WasmEntrypoint` is defined in a different file from where `entrypoint_node_create` is called?
In this case it would be hard to debug such error

Reply:
that trait is not meant to be implemented manually, and even if it were, you would get a regular error from Rust telling you that it's implemented twice, and where the implementations are, so I think it's fine

Reply:
I meant that `entrypoint_command_handler` will be called in a different crate (and thus the implementation)

Reply:
I think even in that case, normal `impl trait` visibility rules should still apply, although I am not 100% sure it's what you are referring to